components/create-state.js

```diff
@@ -0,0 +1,28 @@
+/**
+ *  file: create-state.js
+ */
+
+var Lm = require('./Lm');
```

Comment:
If we're switching to a lower case convention (as discussed with James today), maybe this should now be createLm.